docs/solana/migration/ledger.mdx

```diff
@@ -62,6 +63,12 @@ Your public key will change on Solana but the private key will remain the same.
 - [Solana Ledger App](https://support.ledger.com/hc/en-us/articles/360016265659-Solana-SOL-?support=true)
   using Ledger Live
 
+:::note Close Ledger Live
+
+Before starting these steps, close the Ledger Live application or the migration will fail.
```

Comment:
```suggestion
Before starting these steps, close the Ledger Live application or the transactions will fail.
```
I'd like to frame this around the specific action and not the more nebulous "migration"